### PR TITLE
feat: add information pagination

### LINKED ISSUE #44: do not leave useless comment
I don't like the comment with debug code and other something not useful info in our codes，Please delete these lines.

https://github.com/deamgo/uipaas-home/blob/1e6aa5e4583154a667ccb0b86f705ef885dbd4b7/backend/api/user/user.go#L34C47-L34C47

### PR DESCRIPTION
**Pull request by oteg006**
### Description

Add company information pagination display
fixes #44
impl  #9 (issue numbers)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible

### CODE REVIEW

**Comment by yuluo-yx** [comment]
pls update pr titile @oteg006

**Comment by oteg006** [comment]
> pls update pr titile @oteg006

it is done

File: backend/api/companyinfo/companyinfo.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	"strconv"
+)
+
+const (
+	GetInfoListSuccess = 0
+	GetInfoListFailed  = -1
+)
+
+type CompanyInfoGetReq struct {
+	// 分页参数
```
[the remark this answers is not in the document]
Comment (oteg006) [reply backend/api/companyinfo/companyinfo.go]: ok

File: backend/db/db.sql [hunk 2]
```diff
@@ -6,3 +6,24 @@ CREATE TABLE IF NOT EXISTS user (
...
+insert into user values (1,'admin','','admin');
+
+
+
+
+
+
+
+
+
```
[the remark this answers is not in the document]
Comment (oteg006) [reply backend/db/db.sql:29]: ok

**Review by zccabb — APPROVED** [review]

File: backend/api/companyinfo/companyinfo.go [hunk 3]
```diff
@@ -0,0 +1,78 @@
+package companyinfo
```
Comment (yanbingbing) [inline backend/api/companyinfo/companyinfo.go:1]: does this package name & file name & folder name obey our rules？

File: backend/api/companyinfo/companyinfo.go [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+import (
+	"github.com/deamgo/uipaas-home/backend/context"
+	"github.com/deamgo/uipaas-home/backend/dao"
+	"github.com/deamgo/uipaas-home/backend/pkg/log"
+	"github.com/deamgo/uipaas-home/backend/pkg/types"
+	"github.com/deamgo/uipaas-home/backend/service/companyinfo"
+	"github.com/gin-gonic/gin"
+	"go.uber.org/zap"
+	"net/http"
+	"strconv"
```
Comment (zccabb) [inline backend/api/companyinfo/companyinfo.go:12]: sorted imports.

File: backend/api/companyinfo/companyinfo.go (hunk 3 above)
Comment (yanbingbing) [reply backend/api/companyinfo/companyinfo.go:1]: @yuluo-yx

Comment (yuluo-yx) [reply backend/api/companyinfo/companyinfo.go:1]: > @yuluo-yx

okay. The naming rules here do not comply with our code specifications. Please refer to the code specification document for modification.

Comment (yuluo-yx) [reply backend/api/companyinfo/companyinfo.go:1]: > > @yuluo-yx
> 
> okay. The naming rules here do not comply with our code specifications. Please refer to the code specification document for modification. @oteg006

Comment (oteg006) [reply backend/api/companyinfo/companyinfo.go:1]: > > @yuluo-yx
> 
> okay. The naming rules here do not comply with our code specifications. Please refer to the code specification document for modification. @oteg006

ok

**Review by zccabb — APPROVED** [review]

**Pull request merged**